test/check-application

```diff
@@ -672,6 +672,54 @@ class TestNavigator(testlib.MachineCase):
         self.assertEqual(m.execute("ls -l /home/admin/newdir/file1")[:10], "-rwxrwxrwx")
         self.assertEqual(m.execute("ls -l /home/admin/newdir/file2")[:10], "-rwxrwxrwx")
 
+    def testKeyboard(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_dir("/home/admin")
+
+        self.enter_navigator()
+
+        # FIXME: Use a loop here? (running m.execute inside for loop doesn't work)
+        m.execute("touch /home/admin/file1 /home/admin/file2 /home/admin/file3 /home/admin/file4 /home/admin/file5 /home/admin/file6 /home/admin/file7")
```

Comment:
you can loop in bash:

```
for i in {1..10}; do echo $i; done
```
However this is bash-ism, might not work in Ubuntu as it uses dash.